rust/operator-binary/src/hdfs_controller.rs

```diff
@@ -48,6 +48,9 @@ use stackable_operator::{
     time::Duration,
     utils::cluster_info::KubernetesClusterInfo,
 };
+use stackable_operator::{
+    k8s_openapi::api::core::v1::ServiceAccount, kube::runtime::events::Recorder,
+};
```

Comment:
Can be merged into existing import above.

Reply:
done